### PR TITLE
[BAC-1089] remove unused opensea methods

### PR DESCRIPTION
**Pull request by SaltyCucumber**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "etherspot",
-  "version": "1.19.0",
+  "version": "1.19.1",
```
Comment (jsidorenko) [inline package.json:3]: 1.20.0 ?

Comment (SaltyCucumber) [reply package.json:3]: thought it's a patch-like change

Comment (jsidorenko) [reply package.json:3]: I would rather say it's a BC :)

Comment (SaltyCucumber) [reply package.json]: No one used it, so technically no harm done

**Review by jsidorenko — APPROVED** [review]

**Pull request merged**